.github/workflows/ci.yml

```diff
@@ -132,8 +132,8 @@ jobs:
         target:
           - i686-unknown-linux-gnu
           - powerpc-unknown-linux-gnu
-#          - powerpc64-unknown-linux-gnu
-          - mips-unknown-linux-gnu
+          - powerpc64-unknown-linux-gnu
+#          - mips-unknown-linux-gnu
```

Comment:
Is MIPS broken for the latest smol version?

Reply:
See https://github.com/async-rs/async-std/actions/runs/7005887907/job/19056537762?pr=1069